core/ui/src/main/java/com/tangem/core/ui/components/bottomsheets/TangemBottomSheetConfigContent.kt

```diff
@@ -1,6 +1,9 @@
 package com.tangem.core.ui.components.bottomsheets
 
+import androidx.compose.runtime.Immutable
+
 /**
  * General interface for bottom sheet config model
  */
+@Immutable
 interface TangemBottomSheetConfigContent
```

Comment:
?
```suggestion
interface TangemBottomSheetContentConfig
```

И у меня есть ощущение, что `Config` тут лишний, кажется просто `Content` хвататет

Reply:
По хорошему здесь вообще нет необходимости расширять интерфейс для реализации боттом щита, так как он всего лишь помечает реализации. Если в рамках фичи это имеет смысл, то в рамках ДС компонента нет - наоборот, нужна гибкость и свобода реализации любого удобного для нас контента.

Другой пункт, почему я не хочу переименовывать, это возникновение мердж конфликтов при влитии в дев.

Предлагаю подождать доработки этого компонента.